src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -88,12 +93,17 @@ def initialize_jobs(self):
                 meta=job_meta,
                 parent_job_id=job_input.get("parent_job_id", None),
             )
+            refresh_state = 1 if status not in TERMINAL_STATES else 0
             self._running_jobs[job_id] = {
-                "refresh": 1
-                if status not in ["completed", "errored", "terminated"]
-                else 0,
+                "refresh": refresh_state,
                 "job": job,
             }
+            # if the job is in a terminal state, add it to the _completed_job_states index
```

Comment:
initialisation previously did not include adding jobs to the `_completed_job_states` index, so one had to wait until there was a status update to populate the index properly. This fixes that problem.